solr/solrj/src/java/org/apache/solr/client/solrj/impl/Http2SolrClient.java

```diff
@@ -1096,38 +1096,89 @@ public Builder withTheseParamNamesInTheUrl(Set<String> urlParamNames) {
 
     /**
      * Set maxConnectionsPerHost for http1 connections, maximum number http2 connections is limited
-     * by 4
+     * to 4
+     *
+     * @deprecated Please use {@link #withMaxConnectionsPerHost(int)}
      */
+    @Deprecated(since = "9.2")
     public Builder maxConnectionsPerHost(int max) {
+      withMaxConnectionsPerHost(max);
+      return this;
+    }
+    /**
+     * Set maxConnectionsPerHost for http1 connections, maximum number http2 connections is limited
+     * to 4
+     */
+    public Builder withMaxConnectionsPerHost(int max) {
       this.maxConnectionsPerHost = max;
       return this;
     }
 
+    /**
+     * @deprecated Please use {@link #withIdleTimeout(int)}
+     */
+    @Deprecated(since = "9.2")
     public Builder idleTimeout(int idleConnectionTimeout) {
+      withIdleTimeout(idleConnectionTimeout);
+      return this;
+    }
+
+    public Builder withIdleTimeout(int idleConnectionTimeout) {
       this.idleTimeout = idleConnectionTimeout;
       return this;
     }
 
+    /**
+     * @deprecated Please use {@link #withHttp1_1(boolean)}
+     */
+    @Deprecated(since = "9.2")
     public Builder useHttp1_1(boolean useHttp1_1) {
```

Comment:
I actually think "useHttp1_1" sounds nicer than "with" in this case but if you think is an improvement then fine.

Reply:
humm...    I like useHttp1_1 better than setHttp1_1..   and I don't love `withHttp1_1`...       I think I'll change back!